src/components/PopUpText.tsx

```diff
@@ -47,20 +47,27 @@ const PopUpText: React.FC = () => {
       <div className="orbit_title_popup">
         <div className="orbit_title_info_header">{headerText}</div>
         <div className="orbit_title_info_body" tabIndex={0}>
-          <strong>Distance: </strong>
-          {bodyDistanceText}
-          <br />
-          <strong>About: </strong>
-          {bodyAboutText}
-          <br />
-          <strong>Risks</strong>
-          <RiskIcons key={activeId + "_risk_icon_component"} id={activeId} />
+          <section className="title_info_section">
+            Distance: <span>{bodyDistanceText}</span>
+          </section>
+          <section className="title_info_section">
+            About: <span>{bodyAboutText}</span>
+          </section>
+          <section className="title_info_section">
+            Risks
+            <span>
+              <RiskIcons
+                key={activeId + "_risk_icon_component"}
+                id={activeId}
+              />
+            </span>
+          </section>
```

Comment:
I needed these to be customizable elements. 
<strong> and <br> are not customizable with css.

Reply:
This is outdated because I updated the className to "orbit_title_info_section" instead of  just "title_info_section" to follow the naming convention.